k9mail/src/main/java/security/planck/sync/PlanckSyncRepository.kt

```diff
@@ -116,7 +116,7 @@ class PlanckSyncRepository @Inject constructor(
             }
 
             SyncHandshakeSignal.SyncPassphraseRequired -> {
-                k9.showPassphraseDialogForSync()
+                //k9.showPassphraseDialogForSync()
```

Comment:
no longer needed because we don't use that key for sync?

Reply:
Well I don't know now which behavior we will have for sync, but this does not look right. Good catch, I will check with Dirk.

Reply:
I am assuming that because now we make the check for passphrases to the core actively from the app, we have no callbacks anymore. That sounds about right.

Reply:
ok cool makes sense to me

Reply:
I was wrong, I fixed a bunch of things and also created EFA-606.